holoviews/element/raster.py

```diff
@@ -644,6 +421,8 @@ class RGB(Image):
         If an alpha channel is supplied, the defined alpha_dimension
         is automatically appended to this list.""")
 
+    _vdim_reductions = {1: Image}
```

Comment:
Yes, please file an issue. Definitely want to address this but don't want to make this PR any longer either!